docs/user-docs/latest/media-library/organizing-assets-with-folders.md

```diff
@@ -10,8 +10,9 @@ Folders in the Media Library help you organize uploaded assets. Folders sit at t
 
 From the Media Library, it is possible to view the list of folders and browse a folder's content, create new folders, edit an existing folder, move assets to a folder, and delete a folder.
 
-::: caution
-Downgrading from Strapi v4.3.0-beta to v4.2.0 will delete folders of the Media Library. Even though assets should be preserved, we recommend you test beta versions of Strapi on a new project rather than on an existing one, as unexpected consequences could impact your database.
+::: note
+* Folders are not displayed in the [Content Manager](/user-docs/latest/content-manager/introduction-to-content-manager.md).
+* Folders follow the permission system of assets (see [Users, Roles & Permissions](/user-docs/latest/users-roles-permissions/introduction-to-users-roles-permissions.md)). You can't define specific permissions for a folder.
```

Comment:
Maybe giving an example? `If you can create (upload) a file, you can create a folder. etc.`

Reply:
Thanks! See [here](https://github.com/strapi/documentation/pull/1008#discussion_r928957376)